specs/simple-serialize.md

```diff
@@ -108,7 +108,7 @@ We now define Merkleization `hash_tree_root(value)` of an object `value` recursi
 
 ## Self-signed containers
```

Comment:
@JustinDrake 
Do we still use this title?

Reply:
Yes :)